src/main/java/seedu/iscam/logic/commands/RescheduleMeetingCommand.java

```diff
@@ -55,7 +55,7 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         ObservableList<Meeting> meetings = model.getFilteredMeetingList();
-        if (index.getZeroBased() >= meetings.size()) {
+        if (index.getZeroBased() >= meetings.size() || index.getZeroBased() < 0) {
```

Comment:
Perhaps checking whether index < 0 is unnecessary because the error would be thrown from parseIndex method in ParseUtil class?

Reply:
Good observation, I will be removing that check condition then.